sympy/physics/units/tests/test_dimensions.py

```diff
@@ -73,3 +84,6 @@ def test_mul_div_exp():
 
     assert length != 1
     assert length / length != 1
+
+    length_0 = length ** 0
+    assert length_0.get_dimensional_dependencies() == {}
```

Comment:
Can you show me some real life use-case? It looks like you would like to test the internals of `_get_dimensional_dependencies_for_name`.